Packs/GuardiCore/Integrations/integration-GuardiCore.yml

```
@@ -728,6 +728,27 @@ script:
         return entry;
     }
 
+    var request_logout = function(method, url, token){
```

js, not python, let's conform.
```suggestion

    var requestLogout = function(method, url, token) {
```